pkg/combustion/templates/15-rke2-installer.sh.tpl

```diff
@@ -6,10 +6,8 @@ mkdir -p /var/lib/rancher/rke2/agent/images/
 cp {{ .ImagesPath }}/* /var/lib/rancher/rke2/agent/images/
 umount /var
 
-{{- if .ConfigFile }}
```

Comment:
I imagine most users will want to do _some_ sort of customization to their RKE2 instance and not be comfortable with the install out of box, but why is it now a requirement?

Reply:
It is a requirement because even if a file is not provided, we store one with a default value. Currently it is only `cni: cilium` in these cases but it can easily grow.